app/components/treatment/Treatment.js

```diff
@@ -98,7 +106,7 @@ export class Treatment extends Component {
 
 // -=-=-=-=-=-= CONTAINER =-=-=-=-=-=-
 
-const mapState = ({ user, treatment }) => ({ user, treatment })
+const mapState = ({ user, plan }) => ({ user, plan })
```

Comment:
actually read this https://github.com/jakepeyser/steps/pull/116#discussion_r89681450 as an alternative